### PR TITLE
Update pricing of exceeded bandwidth usage

### PR DESCRIPTION
**Pull request by muhammetssen**

### CODE REVIEW

File: public/faq/redis.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
...
-      "answer": "For pay is you go plan, it is free up to monthly bandwidth limit of 200GB. Beyond that, we charge $0.03 for each additional GB data transfer.\nFor Pro and Enterprise plans, it is charged at a rate of $0.03 per GB for all the usage throughout the month. For use cases with high volume, you may consider VPC Peering which minimizes the data transfer cost. Contact us at support@upstash.com for details.\nPrice can change depending on cloud provider's fee for the incoming traffic. $0.03 is when the client is in the same region."
+      "answer": "For pay is you go plan, it is free up to monthly bandwidth limit of 200GB. Beyond that, we charge $0.03 for each additional GB data transfer.\nFor Pro and Enterprise plans, it is charged at a rate of $0.03 per GB for all the usage throughout the month up to the plan's limit. Any usage beyond that is charged at a rate of $0.10 per GB. For use cases with high volume, you may consider VPC Peering which minimizes the data transfer cost. Contact us at support@upstash.com for details.\nPrice can change depending on cloud provider's fee for the incoming traffic. $0.03 is when the client is in the same region."
```
Comment (sertugkaya) [inline public/faq/redis.json:33]: "VPC Peering" => "VPC Peering or AWS Privatelink"

**Review by sertugkaya — COMMENTED** [review]
left a comment.

**Comment by sertugkaya** [comment]
@enesakar can you also review pls

**Comment by enesakar** [comment]
can you also update this tooltip

CleanShot 2024-11-17 at 22 48 27@2x

File: public/faq/redis.json (hunk 1 above)
Comment (muhammetssen) [reply public/faq/redis.json:33]: ✅

**Review by sertugkaya — APPROVED** [review]

**Pull request merged**